src/app/checkout/Checkout.tsx

```diff
@@ -179,7 +180,16 @@ class Checkout extends Component<CheckoutProps & WithCheckoutProps & WithLanguag
                 this.handleReady();
             }
         } catch (error) {
-            this.handleUnhandledError(error);
+            if ( error.status === 500 && error.message === 'Tax Service Unavailable' ) {
+                this.handleUnhandledError(new CustomError({
```

Comment:
So that tax error can pop up at any point, not just the initial load, e.g.: submit order, setting shipping address etc.... What I'd do is modify `handleUnhandledError` to do the error handling instead, because that method is triggered whenever there's an exception that's not caught at the layers below.

Also, I'd have the SDK throw a typed error, e.g.: `TaxServiceUnavailableError` and you can catch it directly rather than having to check the exact error message, which is comparatively more brittle.